documents/Final Report/reflection.tex

```diff
@@ -2,14 +2,15 @@ \chapter{Reflection on the product and process}
-The positive outcomes of the product is that we have implemented some features that were not planned. Selling land is a feature that we implemented, but was not planned beforehand. With selling land we earn money with land we do not use. 
+The product in its entirely as we have now, is not exactly as we had planned at the beginning of the project. There are some positive and negative outcomes of what we had planned to achieve. The negative outcome is that we are missing some decision logic that we had planned in the beginning of the project. We wanted to have more interaction with other agents, especially the buying and selling of land is devious. 
+The positive outcome of the product is that we have implemented some features that were not planned. Selling land is a feature that we implemented, but was not planned beforehand. With selling land we earn money with land we do not use.
 
 \section{Reflection from the software engineering perspective}
 We have experienced the sprints and communication in our own group as adequate. We had some starting problems, but with the grades of the first weeks, we realized that this was not the way to do a good context project. After sprint two, we made clear arrangements about the project and the collaboration as a group. In our experience, this worked out well, which also can be seen in our rising grades of the sprints as well as the CEQ.
 \\\\
 But the biggest issue we had in this project was the communication between groups. This have caused some delay. In the first few weeks, we had only one meeting with all the groups together. The rest of the time, we worked with our own group together and communication between groups was done without face to face communication. This resulted in a very late response time for small issues, which caused some process loss. After sprint 5, we had two meetings every week and made some clear arrangements about responding to pull requests and issues about the connector.
 \\\\
-In retrospect we should have immediately started with figuring out how the environment worked and what its limitations were. If we had this in mind, it was easier to accomplish our final result that we had planned. We should also have asked more questions to other groups and people with more knowledge about the connector from the beginning of the project. After week 4, this was improved and we started to be more productive. 
+What we should have done in another way is that we should have started immediately with figuring out how the environment worked and what its limitations were. If we had this in mind, it was easier to accomplish our final result that we had planned. We also should have asked more questions to other groups and people with more knowledge about the connector from the beginning of the project. After week 4, this was improved and we started to be more productive
```

Comment:
*in a different way
*if we would have had
*it would have been

Missing a . after productive?